src/components/TimeTravel/_TimelineRange.js

```diff
@@ -1,7 +1,18 @@
 import React from 'react';
 import moment from 'moment';
+import styled from 'styled-components';
 import PropTypes from 'prop-types';
 
+const TimelineRangeOverlay = styled.div.attrs({
+  style: ({ position }) => ({ transform: `translateX(${position}px)`})
```

Comment:
`position` feels a bit ambiguous vs `x`. and I get `position` confused w/ the other css stuff.

and does this need to be a style attr?

Reply:
Makes sense!

> does this need to be a style attr?

Moving values that change a lot from component styles into `attr.style` is an optimization that _styled-components_ recommend, to prevent recreating the whole styled component object every time e.g. a position or size of an object changes.